packages/babel-plugin-named-asset-import/package.json

```diff
@@ -13,5 +13,16 @@
   ],
   "peerDependencies": {
     "@babel/core": "^7.1.0"
+  },
+  "devDependencies": {
+    "@babel/preset-env": "^7.1.0",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^23.6.0",
```

Comment:
We can probably just drop all the babel necessity and use `require()` for the tests.